service/src/main/java/bio/terra/drshub/models/DrsApi.java

```diff
@@ -13,4 +13,8 @@ public DrsApi(ApiClient apiClient) {
   public void setBearerToken(String bearerToken) {
     ((OAuth) this.getApiClient().getAuthentication("BearerAuth")).setAccessToken(bearerToken);
   }
+
+  public void setHeader(String name, String value) {
```

Comment:
do you mind adding a quick unit test to verify `setHeader`?

Reply:
I started writing this unit test but then I realized there's no way to get the `defaultHeaders` from the ApiClient (it's a private field and there's no getter). Do you have any suggestions on how to verify that the header was actually set in the test?

Reply:
Hmm I took a look and I don't see a very easy way to do this, unless we want to add a getter for it. I think I'd be ok omitting this test as long as we're confident that it's tested on the TDR side.